### ISSUE TITLE
[lld] version mismatch with llvm

### ISSUE THREAD

**Issue by blu** [skycocker/chromebrew#2252]
On aarch64: current lld is from llvm 5, whereas the llvm package is 6. As a result lld cannot run due to missing versioned SOs:

```
$ ldd `which lld`
        libLLVMSupport.so.5 => not found
        libpthread.so.0 => /usr/local/bin/../lib/libpthread.so.0 (0xecbdc000)
        liblldDriver.so.5 => /usr/local/bin/../lib/liblldDriver.so.5 (0xecbba000)
        liblldCOFF.so.5 => /usr/local/bin/../lib/liblldCOFF.so.5 (0xecb37000)
        liblldELF.so.5 => /usr/local/bin/../lib/liblldELF.so.5 (0xec95f000)
        libstdc++.so.6 => /usr/local/bin/../lib/libstdc++.so.6 (0xec80c000)
        libm.so.6 => /lib/libm.so.6 (0xec794000)
        libgcc_s.so.1 => /usr/local/bin/../lib/libgcc_s.so.1 (0xec768000)
        libc.so.6 => /usr/local/bin/../lib/libc.so.6 (0xec67b000)
        /lib/ld-linux-armhf.so.3 (0xecbff000)
        liblldConfig.so.5 => /usr/local/bin/../lib/../lib/liblldConfig.so.5 (0xec669000)
        liblldMachO.so.5 => /usr/local/bin/../lib/../lib/liblldMachO.so.5 (0xec5db000)
        liblldCore.so.5 => /usr/local/bin/../lib/../lib/liblldCore.so.5 (0xec5a2000)
        liblldReaderWriter.so.5 => /usr/local/bin/../lib/../lib/liblldReaderWriter.so.5 (0xec58c000)
        liblldYAML.so.5 => /usr/local/bin/../lib/../lib/liblldYAML.so.5 (0xec57b000)
        libLLVMObject.so.5 => not found
        libLLVMOption.so.5 => not found
        libLLVMSupport.so.5 => not found
        libLLVMAArch64CodeGen.so.5 => not found
        libLLVMAArch64AsmPrinter.so.5 => not found
        libLLVMAArch64AsmParser.so.5 => not found
        libLLVMAArch64Desc.so.5 => not found
        libLLVMAArch64Disassembler.so.5 => not found
        libLLVMAArch64Info.so.5 => not found
        libLLVMAArch64Utils.so.5 => not found
        ...
```

**Comment by blu** [comment]
Same issue with package clang.

**Comment by lovedheart** [comment]
@blu I am fixing it.

**Comment by lovedheart** [comment]
@blu This should be fixed by #2253
